apps/client/src/app/core/scheme/indicators/indicators.component.ts

```diff
@@ -0,0 +1,277 @@
+
+      accept: () => {
+        this.toast.show('info', 'Eliminando..', 'Eliminando indicador..')
+        this.onDelete(id)
+      },
+      reject: () => {
+        this.toast.show('error', 'Rechazado', 'Haz rechazado la eliminación')
+      }
+    })
+  }
+  getAll() {
+    this.indicatorService.getAll(this.paginated).subscribe({
+      next: (res) => {
+        if (res.status === 'success') {
+          this.totalItems = res.data.totalItems
+          this.schemes = res.data.items.map((indicator) => {
+            return {
+              ...indicator,
+              categories: [],
+              criterias: []
+            }
+          })
+        }
+        this.isFetching = false
+        for (const scheme of this.schemes) {
+          this.getCategoriesPerIndicator(scheme.index)
+          this.getCriterionPerIndicator(scheme.index)
+        }
+      },
+      error: (e) => {
+        console.error(e)
+        this.toast.show('error', 'Error', e.error.data.message)
+        this.isFetching = false
+      }
+    })
+  }
+
+  getCategoriesPerIndicator(indicatorId: number) {
+    this.categoryService
+      .getAllCategoriesByIndicatorIndex(indicatorId)
+      .subscribe({
+        next: (res) => {
+          if (res.status === 'success') {
+            this.schemes.map((scheme) => {
+              if (scheme.index === indicatorId) {
+                scheme.categories = res.data
+              }
+            })
+          }
+        },
+        error: (e) => {
+          console.error(e)
+          this.toast.show('error', 'Error', e.error.data.message)
+        }
+      })
+  }
+
+  getCriterionPerIndicator(indicatorId: number) {
+    this.criteriaService
+      .getAllCriterionByIndicatorIndex(indicatorId)
+      .subscribe({
+        next: (res) => {
+          if (res.status === 'success') {
+            this.schemes.map((scheme) => {
+              if (scheme.index === indicatorId) {
+                scheme.criterias = res.data
+              }
+            })
+          }
+        },
+        error: (e) => {
+          console.error(e)
+          this.toast.show('error', 'Error', e.error.data.message)
+        }
+      })
+  }
+
+  onCreate() {
+    this.fetchCreateIndicator = true
+    const { index, name, alias, helpText } = this.formGroup.controls
+    if (!index.value || !name.value || !alias.value || !helpText.value) return
+    const indicator: Indicator = {
+      index: index.value,
+      name: name.value,
+      alias: alias.value,
+      helpText: helpText.value
+    }
+    this.indicatorService.create(indicator).subscribe({
+      next: () => {
+        this.fetchCreateIndicator = false
+        this.getAll()
+        this.toast.show('success', 'Creado', 'Indicador creado con éxito')
+        this.closeDialog()
+      },
+      error: (e) => {
+        this.fetchCreateIndicator = false
+        this.toast.show('error', 'Error', e.error.data.message)
+        this.closeDialog()
+      }
+    })
+  }
+
+  onEdit() {
```

Comment:
No estás chequeando que el form sea válido

```suggestion
  onEdit() {
    if (this.formGroup.invalid) return
    ...
```